lib/evm/program_counter.ex

```diff
@@ -0,0 +1,32 @@
+defmodule EVM.ProgramCounter do
+  @doc """
```

Comment:
Can you add a quick module doc?

Reply:
Oh yeah, good call: [Add @moduledoc to the `ProgramCounter` module](https://github.com/exthereum/evm/pull/23/commits/794e4892c490c815d6bb08ffa1bd4d0e43ec9f3a)